Makefile

```diff
@@ -47,7 +54,7 @@ ifeq ($(USE_IMAGE_DIGESTS), true)
 endif
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= $(IMAGE_TAG_BASE):latest
```

Comment:
while at it, could you please rename this to OPERATOR_IMG ? There will be other images as well. I am especially thinking of publishing bundles which takes us one step closer to having (or not having) a catalog.